examples/swash_render/src/main.rs

```diff
@@ -84,7 +84,7 @@ fn main() {
             ..Default::default()
         };
 
-        let mut builder = layout_cx.tree_builder(&mut font_cx, display_scale, &root_style);
+        let mut builder = layout_cx.tree_builder(&mut font_cx, display_scale, true, &root_style);
```

Comment:
Can we come up with some solution to make it clearer what this boolean parameter is? Probably either an options struct that wraps it or a `/* quantize */` comment.